.circleci/config.yml

```
@@ -294,6 +294,12 @@ jobs:
           command: yarn test:stress
           no_output_timeout: 10m
       - upload_artifacts
+      - store_artifacts:
+          path: /home/circleci/project/packages/server-wallet/profiling-data/stress-test-1-payment.html
+          destination: stress-test-1-payment.html
+      - store_artifacts:
+          path: /home/circleci/project/packages/server-wallet/profiling-data/stress-test-25-payment.html
+          destination: stress-test-25-payment.html
```

```suggestion
      - store_artifacts:
          path: /home/circleci/project/packages/server-wallet/profiling-data
```

We used to have a mess of uploading a dozen different artifacts. It's much cleaner to just upload an entire directory.

(Even better might be to just put the artifacts in the `artifacts` folder that is currently uploaded?)

---

There is also captured raw data in `profiling-data` that we don't want to upload but just storing the html files in the artifact folder makes sense.